.docs/user-guide/config.md

```diff
@@ -256,3 +256,22 @@ supported:
 Docker   | docker
 
 The volume driver `docker` is automatically activated.
+
+### VOLUME MOUNT PRE-EMPTION
+There is a capability to pre-emptively detach any existing attachments to other
+instances before attempting a mount.  This will enable use cases for
+availability where another instance must be able to take control of a volume
+without the current owner instance being involved.  The operation is considered
+equivalent to a power off of the existing instance for the device.
+
+Property Name | Environment Variable | CLI Flag
+--------------|----------------------|-------------
+`volume.mountpreempt`   | `REXRAY_VOLUME_MOUNTPREEMPT`          | `--preempt`
```

Comment:
Hi @clintonskitson,

Wouldn't the property name and CLI flag be `rexray.volume.mountpreempt` and `rexrayPreempt` (respectively)? Also, if you note, I actually no longer use the tables elsewhere to show variations of the name but reference the Configuration section with a link so readers can see how the names are derived from a single property name.